### PR TITLE
Improve plan file layout and update/remove old comments

### PR DESCRIPTION
**Pull request by alexbrand**
There were some comments that were outdated. I updated them and also removed some comments that seemed unnecessary. Also made some cosmetic changes to improve the file's layout and spacing.

### CODE REVIEW

**Comment by dkoshkin** [comment]
What would this look like if we added a newline after each section.

I thin adding a new line [here](https://github.com/apprenda/kismatic/blob/plan-file-comments/pkg/install/plan.go#L174) and removing `\n` from [here](https://github.com/apprenda/kismatic/blob/plan-file-comments/pkg/install/plan.go#L205)

**Comment by alexbrand** [comment]
Added a change to include a new line before a new "section".

File: pkg/install/test/plan-template-with-storage.golden.yaml [hunk 1]
```diff
@@ -6,12 +6,13 @@ cluster:
   admin_password: password
 
   # When true, installation will not occur if any node is missing the correct
```
Comment (dkoshkin) [inline pkg/install/test/plan-template-with-storage.golden.yaml]: To be consistent should we set this to something like
> Set to true if nodes have the required packages installed, installation will not occur if any node is missing the correct deb/rpm packages.

Comment (dkoshkin) [reply pkg/install/test/plan-template-with-storage.golden.yaml]: Also any other `When true` to be consistent with `Set to...`

Comment (dkoshkin) [reply pkg/install/test/plan-template-with-storage.golden.yaml]: Should we remove `you/your` and use `the` or `cluster` etc.

Comment (alexbrand) [reply pkg/install/test/plan-template-with-storage.golden.yaml]: I was using you based on this style guide: https://kubernetes.io/docs/home/contribute/style-guide/#address-the-reader-as-you

Was there any specific line that you think would benefit from not using `you`?

Comment (dkoshkin) [reply pkg/install/test/plan-template-with-storage.golden.yaml]: Actually I didnt realize that was in the style-guide, ignore that.

**Comment by alexbrand** [comment]
Updated incosistent "When true, ..." comments.

**Comment by dkoshkin** [comment]
LGTM

**Pull request merged**